<a href="https://colab.research.google.com/github/rishika-p/assignment-1/blob/master/q3.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [5]:
a=input("Enter type of seat: ")
b=input("Enter payment mode:")
if a=="stalls":
  if b=="cash":
    print(625-(625*0.1))
  elif b=="credit":
    print(625-(625*0.05))
elif a=="circle":
  if b=="cash":
    print(750-(750*0.1))
  elif b=="credit":
    print(750-(750*0.05))
elif a=="upperclass":
  if b=="cash":
    print(850-(850*0.1))
  elif b=="credit":
    print(850-(850*0.05))
elif a=="box":
  if b=="cash":
    print(1000-(1000*0.1))
  elif b=="credit":
    print(1000-(1000*0.05))


Enter type of seat: circle
Enter payment mode:cash
675.0


ENTERTAINMENT PARADISE::::
TICKET RATES FOR STALLS:625
FOR CIRCLE:750
FOR UPPERCLASS:850
FOR BOX:1000
IF TICKETS ARE PURCHASED ON CASH DISCOUNT IS 10% IN CASE BY CREDIT CARD THEN DISCOUNT IS 5%
I/P:TYPE OF SEAT:CIRCLE;
PAYMENT MODE:CASH;
0/P:675